doc/getting_started/toolchain_other_x_compilers.rst

```diff
@@ -29,6 +29,16 @@ Follow these steps to use one of these toolchains.
       # On Fedora or Red Hat
       sudo dnf install arm-none-eabi-newlib
 
+#. Install a C library suitable for the target system.
+
+   For example, you might install the picolibc-arm-none-eabi package
+   on Debian-based Linux systems:
+
+   .. code-block:: console
+
+      # On Debian or Ubuntu
+      sudo apt-get install picolibc-arm-none-eabi
```

Comment:
Right, it's only available in the development version of Ubuntu: https://bugs.launchpad.net/ubuntu/+source/picolibc
We could write some instructions for getting picolibc installed from source, or I could create pre-built binaries against the current ARM embedded toolchain as I do for the current RISC-V embedded toolchain:
https://github.com/keith-packard/picolibc/releases/tag/1.4.6

Reply:
I think that until picolibc is part of the stable software repositories (debian/Ubuntu) or Zephyr SDK, then I think it is important to include a bit more detailed description on how to obtain pre-built packages or how to build from source.

Just curious, can picolibc be used with `armgcc` on windows ?
I see it can be built and used natively on posix systems (which windows is not), but I don't see if it can be built and used by a cross compiler on windows (such as armgcc).

Reply:
I haven't ever tried using armgcc on windows to build picolibc, but I don't know of any reason it *wouldn't* work, aside from GCC-isms in the code. picolibc inherits a lot of newlib code, and that does end up depending on GCC. Probably worth a try at least.

Reply:
I got picolibc compiling using armgcc under Windows over the weekend; meson needed mingw and msys to get the necessary tooling. I'll have patches merged to picolibc for the next release.